### DISCUSSION TITLE
Error 500: QFieldCloud is down on 2022-07-04 since 11:46 UTC until 22:19 UTC

### DISCUSSION THREAD

**Discussion by suricactus** [opengisch/QField#3050] [Bugs]
We are aware QFieldCloud services are not functioning properly due to external factors that are not under OPENGIS.ch developers control. Connectivity to our main database is higly limited and causes HTTP 500 errors. We are doing our best to restore the service as soon as possible. 

Will keep you updated once the service is restored.

**Comment by JUAN1936** [comment]
Gracias

**Comment by suricactus** [comment]
As of 22:19 UTC we are finally online again. Sorry for the inconvinience, the services should work as expected now.

**Comment by AscencioLadino** [comment]
Thank you very much, works fine

**Comment by JUAN1936** [comment]
thank you very much for the incredible work
